site/homepage/content/release-notes.md

```diff
@@ -148,6 +150,14 @@ title = "Release Notes"
   properties instead of working on given ones. The new method name now more accurately conveys
   what the method is used for. The same change has been applied to the other `customize[*]Config`
   methods in the `AbstractAdapterConfig` class.
+* The file based as well as the MongoDB based device registry implementations now disallow certain
+  special characters in device and tenant identifiers. The configuration properties
+  `HONO_REGISTRY_HTTP_TENANT_ID_PATTERN` and `HONO_REGISTRY_HTTP_DEVICE_ID_PATTERN` (and corresponding ones with
+  prefix `HONO_REGISTRY_REST` for the file based registry) can be used to override the patterns for matching
+  valid identifiers. Please refer to the 
+  [file based registry]({{% doclink "/admin-guide/file-based-device-registry-config/#service-configuration" %}}) or
+  [MongoDB based registry]({{% doclink "/admin-guide/mongodb-device-registry-config/#service-configuration" %}})
+  configuration guide for details.
```

Comment:
is this supposed to be documenting the status quo of the 1.3.0 release only? My understanding is that you want to remove these restrictions again in 1.5.0, 1.4.1 and 1.3.1. I guess we then should also add a corresponding entry to the *API Changes* section for the upcoming 1.5.0, right?

Reply:
This shall document the fact that there are restrictions on the allowed characters now, not talking about the colon in particular. So this is still valid after the patch releases.

> I guess we then should also add a corresponding entry to the API Changes section for the upcoming 1.5.0, right?

I've added an entry there.